.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/aerogear/mobile-security-service
+
+    docker:
+      - image: circleci/golang:1.10
```

Comment:
changed to 1.11 and asked Wei if there is a standard in Mobile Services

Reply:
changing to 1.10 as this is what is used in productisation.

Reply:
Hmm, 1.10 in nearly a year old. Is it not a good idea to use 1.11 while we have the chance and nothing depending on this project? Or is it because it depends on other parts of Mobile Services?

Reply:
@craicoverflow We certainly could, as there is a very small chance of an issue arising when it comes time to productisation, however, to be 100% sure there won't be any problems, we are better off staying consistent with the version that it will have to be built with when getting packaged for our customers.